### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by dwillmer**
- [X] Consistent use of single/double quotes. Standalone strings were mostly single-quoted, so i stuck with that, whereas docstrings were all double-quoted, so I left them alone.
- [X] Consistent use of ipython copyright header.
- [X] Re-jig any code over 79 chars, except regex lines.
- [X] Remove triple-quoted strings where the string fits on one line.
- [X] Assign repeated code fragments to local variables, where it makes sense.
- [X] Fix typos.
- [X] Consistent spacing around binary operators.
- [X] Remove mutable default arguments.
- [X] Remove whitespace at end-of-line.
- [X] Factor out complex (and long) if-criteria in pickleutil.
- [X] Move global var in test module to function local.
- [X] Consistent UTF-8 encoding notice in each file (required to keep py2.7 tests passing)

### CODE REVIEW

**Comment by takluyver** [comment]
Thanks for taking the time to work on this. However, we don't generally accept big 'cleanup' PRs - it's liable to cause merge conflicts, it makes git blame less useful, and code style just isn't all that important to us.

It's fine to clean up a specific piece of code you're making functional changes to, but our policy is to reject PRs that just do cleanup, especially when they touch many files at once. Sorry this wasn't clear before you started.

**Comment by ellisonbg** [comment]
I think a blanket rejection is too harsh. This PR does more than simple white space cleanup. And some of that is important refactoring. I agree that the broad code formatting stuff should go in, but the refactoring stuff should.

**Comment by ellisonbg** [comment]
Oops typo - I agree with Thomas that the simple code formatting stuff - white space quotes. - should not go in

File: ipykernel/inprocess/manager.py [hunk 1]
```diff
@@ -23,7 +24,8 @@ class InProcessKernelManager(KernelManager):
     kernel = Instance('ipykernel.inprocess.ipkernel.InProcessKernel',
                       allow_none=True)
     # the client class for KM.client() shortcut
-    client_class = DottedObjectName('ipykernel.inprocess.BlockingInProcessKernelClient')
+    name = 'ipykernel.inprocess.BlockingInProcessKernelClient'
```
Comment (takluyver) [inline ipykernel/inprocess/manager.py]: It doesn't seem like this should be a `name` attribute on the kernel manager class.

**Comment by takluyver** [comment]
Fair enough, I didn't notice that were actual significant changes amongst all the code style stuff. Scanning through the changes, though, I suspect it's going to be quicker to redo those changes starting from master than to separate them out of the formatting changes in this PR.

**Comment by Carreau** [comment]
> Fair enough, I didn't notice that were actual significant changes amongst all the code style stuff. Scanning through the changes, though, I suspect it's going to be quicker to redo those changes starting from master than to separate them out of the formatting changes in this PR.

I seem to agree, and i have a hard time finding the actual changes, even with `?w=0`.

**Comment by Carreau** [comment]
> And some of that is important refactoring

Ok, I'm just spend 30 minutes going through that chuck by chunk, I honestly haven't found the parts that is "important refactoring" and will be needed later. I would be happy to get some pointer to where it is.
I might be tired and missed it though, it's hard to find in this 1k line change.

**Pull request closed without merging**